common.h

```diff
@@ -378,7 +378,7 @@ namespace fsgrids {
    };
    
    struct technical {
-      int sysBoundaryFlag;  /*!< System boundary flags. */
+      uint sysBoundaryFlag;  /*!< System boundary flags. */
```

Comment:
This is the only part of this PR that I don't consider completely trivial. But since sysBoundaryFlag anyway can only take enum values, there is no reason to assume it would ever be negative.